<html>
<head>
<!--center><h2> About Me</ih2></center-->
    <meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="chrome=1">
<meta name="HandheldFriendly" content="True">
<meta name="MobileOptimized" content="320">
<meta name="viewport" content="width=device-width, initial-scale=1.0">
<meta name="referrer" content="no-referrer">

<meta name="description" content="Seke Keretsu.">



<style>
div {
  max-width: 1000px;
  min-width: 100px;
  background-color:#D3D3D3;
  text-align:justify;	
  text-indent:50px; 
  margin-right: 20px;
  margin-left:  20px;
}
</style>


</head>
<center>

<body>
<div>
<style>
.topnav {
  overflow: hidden;
  background-color: #333;
}

.topnav a {
  float: left;
  color: #f2f2f2;
  text-align: center;
  padding: 14px 16px;
  text-decoration: none;
  font-size: 17px;
}

.topnav a:hover {
  background-color: #ddd;
  color: black;
}

.topnav a.active {
  background-color: #4CAF50;
  color: white;
}
</style>
</head>
<body>

<div class="topnav">
  <a href="index.html">Home</a>
  <a class="active "href="research.html">Research</a>
  <a href="publication.html">Publication</a>
  <a href="software.html">Software</a>
  <a href="tutorial.html">Tutorial</a>
   <a href="contact.html">Contact</a>
</div>

<!--p style="text-align:center; margin-left:50px; ">
<img src="https://sekekeretsu.github.io/headshot.jpg" alt=" headshot" width="400">
</p-->

<div>
<p>
This includes ongoing research projects as well as other interest of mine that I inended to pursue in the near future. This works can be broadly classified into the application and development oriented projects </p>
<p>
This part will be updated from time to time 
</p>

Recent studies on GPCR proteins and CRISPR-CAS9 systems have demonstrated that GaMD techniques could be effectively used to study the binding and activation mechanisms of large biomolecular systems. I am curious to find out if these enhanced sampling techniques can be used to resolve existing structural enigmas in the areas of GPCR proteins such as Serotonin receptors.
</p>

    
    
<p>
    Modelling study of Nanobody binding to Angiotensin II Type 1 Receptor (AT1R) and Virtual Screening of Potential AT1R Inhibitors.
Abstract: Angiotensin II type 1 receptors (AT1R) have emerged as an important G Protein-Coupled Receptor (GPCR) because of the role it plays in regulating cardiovascular and renal activities. Consequently, inhibitors that target AT1R are considered as therapeutic agents for heart failure diseases. However, computational studies of the AT1R-G protein interactions have not been studied due to the oversized system that incurs high computational cost. This proposal was designed to study the coupling mechanism of the binding of ligand to AT1R and the conformational changes leading to the stabilization and activation of AT1R using the Gaussian accelerated molecular dynamics (GaMD) simulation. In addition, a docking based virtual screening protocol was also proposed to identify potential AT1R antagonists from the ZINC database. Binding free energy evaluation of selected ligands from the screening will be performed using Molecular Mechanics Poisson-Boltzmann Surface Area (MM-PBSA) method. The outcome of the study could provide useful insight into the mechanism of AT1R activation and may assist in the discovery of AT1R antagonists. 
    </p>
    
<br>
    <p>I have worked in the area of data mining in biological networks under the project entitled, “Protein complex identification in protein-protein interaction networks with the incorporation of gene expression profile,” supervised by Dr.Rosy Sarmah, Tezpur University. The study culminated in the development of new bioinformatics methods that showed statistically better protein complex prediction. The Java implementations of the methods are available at the links below:
http://agnigarh.tezu.ernet.in/~rosy8/shared.html or https://github.com/sekekeretsu/
        </p>
        <p>      
Introduction
Immune evasion via T cell exhaustion, secretion of immunosuppressive mediators and expression of proteins that modulate immune checkpoint, is a well-established hallmark for cancer (1). T cell infiltration to the tumor microenvironment has been considered to be a prerequisite for an efficient response to immunotherapeutic treatment (2,3). Hence, redirecting the immune response to the tumor microenvironment has been considered an important therapeutic route for cancer treatment (2). 
The enzyme cyclic-GMP-AMP synthase (cGAS) detects cytoplasmic self DNA or tumor-derived DNA and generates the secondary messenger cyclic Guanosine Monophosphate–Adenosine Monophosphate (cGAMP) (4). cGAMP binds to the stimulator of interferon genes (STING) and activates it. STING then recruits tank-binding kinase 1 (TBK1) and activates interferon regulatory factor (IRF3) which triggers a cascade of downstream signaling events responsible for activation and infiltration of T cells into the tumor microenvironment and promotes antitumor responses. Hence, STING agonists with the potential to induce immune response are promising drug candidates for cancer (2,5,6). 
The objective of this proposal is to determine the mechanism of STING activation by it endogenous ligand cGAMP at the molecular level and to develop novel STING agonists using state-of-the-art computer-aided drug design (CADD) techniques such as molecular docking, pharmacophore modeling, virtual screening, classical molecular dynamics (MD) and binding energy calculation (7-9). 
Specific Aims
Aim 1: To determine the mechanism of activation of STING by the endogenous cyclic dinucleotide 2’, 3’-cGAMP. 
The stimulator of interferon genes (STING) is a transmembrane protein located in the endoplasmic reticulum (ER). Previous studies have shown how the 2’, 3’-cGAMP interacts with the cytoplasmic domain of STING dimer. However, the mechanism of how 2’, 3’-cGAMP binding leads to activation of STING and the downstream signaling events that follow is unknown. Comparative studies of the STING dimer in apo state and in the 2’, 3’-cGAMP bound state could reveal the essential conformational changes and the key residues that are involved in the activation.
All-atom classical MD simulation of the apo-STING and STING-cGAMP complex (Figure 1B) will be performed to study the dynamic interactions between the STING protein and 2’, 3’-cGAMP at the binding interface and to study the 2’, 3’-cGAMP induced conformational changes in STING. This will be followed by Molecular Mechanics Poisson-Boltzmann Surface Area (MM-PBSA) based binding free energy calculations of the STING-cGAMP complexes with the objective to study the binding affinities between STING and 2’, 3’-cGAMP and to identify residues important for binding with STING (8). The cryo-electron microscopy (cryo-EM) structure of the apo state human STING (PDB 6NT5) and the cGAMP-STING (PDB 4KSY) complex reported by Shang et al., (2019) will be used for the study (10,11). Homology modeling methods will be used to model the missing residues in the protein. All-atom MD simulations and binding energy calculations will be performed using GROMACS and g_mmpbsa package respectively (7,8).


Aim 2: To develop novel hydrolysis resistant nucleotidic STING agonists 
Natural cyclic dinucleotide (CDN) and their derivatives have demonstrated promising antitumor activity and immunostimulating effects both in vitro and in vivo. However, natural CDN and their derivatives are poor pharmaceutical drug candidates due to their instability and high polarity. CDN based compounds consist of two nucleotides that share two phosphodiester bonds and the phosphorus atoms at the phosphodiester links/bonds were negatively charged and were susceptible to hydrolysis by enzymes such as phosphodiesterase  (2,12). 
The CDN 2’, 3’-cGAMP is a heterodimer linked by one 3’-5’-phosphodiester and one 2’-5’-phosphodiester and is the most potent STING agonist in human (Figure 1C). The compound 2’, 3’-cGAMP has a high affinity for human STING with a dissociation constant of 4.59 nM (13). Taking the endogenous cyclic dinucleotide (CDN) cGAMP as an initial structure (13), hydrolysis resistant dinucleotide analogs of cGAMP will be designed to mimic the interactions of cGAMP with STING. Various substitutions will be made for the phosphodiester bond with linker moieties as shown in Figure 1D.  The observations made in Aim 1 will also be used in guiding the process of optimizing the designed compounds. Molecular docking will be performed to study the binding interactions. The binding affinity of the designed compounds will be evaluated iteratively using all-atom classical MD simulations and binding energy calculations (14). 
Aim 3: To discover small molecules from publicly available databases with virtual screening. 
Based on the study from Aim 1, interactions that were significant for the activation of STING will be identified. Pharmacophore models will be developed to search for small molecules that can bind and activate STING. This will be followed by molecular docking of the compounds with STING to evaluate the binding affinity.  The compounds that showed significant binding interactions with STING will be furthered studied using classical MD simulations and free energy calculations (14). The compounds collected from publicly available databases such as the ZINC database (15) or the ChEMBL database (16) will be used for the study. The compounds that showed good binding affinity with binding site residues of STING will be selected as potential STING agonists. These hit compounds maybe studied by experimental collaborators to test its antitumor activity.
Significance
Immunotherapy has emerged as a remarkable pharmaceutical approach to mitigate previously untreatable tumors and metastatic cancers. This was achieved via the immune checkpoint blockade by triggering antitumor T cell response. However, a major drawback of the use of immune checkpoint blockade was in the case of non-immunogenic tumors in which cancer fails to elicit T cell response (2). Earlier studies have shown that the infiltration of T cells into the tumor microenvironment could be an important indicator of an effective response to immunotherapic treatment (3). Hence, efforts to invoke the infiltration of T-cell into the tumor microenvironment could be an effective approach to improve the antitumor activity. 
The cGAS-STING-IRF3 pathway constitutes an important signaling pathway in the innate immune system, protecting the body against various pathogen invasions (Figure 1A) (17). The presence of microbial DNA or self DNA in the cytoplasm represents a danger signal and consequently triggers the innate immune response. The cytoplasmic DNA is detected by the enzyme cGAS which generates the secondary messenger cGAMP (2’, 3’-cGAMP in humans). The cGAMP binds to STING, inducing conformational changes in the STING cytoplasmic domain, which leads to its activation. This is followed by the production of antiviral and pro-inflammatory cytokines such as type 1 interferons (INFs) leading to the infiltration of T-cells into the tumor micro-environment, invoking an immune response. Hence, STING is considered an important target for antitumor drugs and vaccine adjuvants (5,13). 
Great strides have been made in understanding the structural and functional aspects of STING and the components of the innate immune system. However, several aspects of the cGAS-STING-IRF3 signaling mechanism are yet to be uncovered, particularly in understanding how the cGAMP binding leads to activation of the STING. Several nucleotidic and non-nucleotidic STING agonists have been developed over the last few years, some of which have entered clinical trials. The 2’, 3’-cGAMP, which is the endogenous ligand of STING, and its isomers 3’, 3’-cGAMP, 3’, 2’-cGAMP, 2’, 2’-cGAMP were comprehensively studied and tested in animal models for pharmaceutical utility (13,18,19). In addition, various nucleotidic analogs based on c-di-GMP (20,21), c-di-AMP (20,21), etc., were developed, some of which showed high potency for anti-tumor activity. However, the therapeutic utility of dinucleotide analogs has been limited as these compounds have low stability and were susceptible to hydrolysis by various nucleases and phosphodiesterase (12).  Hence, finding nucleotide-based STING agonists with high stability and resistance against hydrolysis poses a critical challenge for therapeutic intervention in cancer treatment.
 
Figure 1. (A)  The cGAS-STING-IRF3 immune signaling pathway (adapted from Ref. (17)) (B) The apo state of the STING homodimer from the cryo-EM (PDB code 6NT5) is shown. The two monomers are shown in cyan and magenta color. The cytoplasmic domain of the cGAMP-STING complex (PDB code 4KSY), shown in gray color, is aligned with the apo-STING structure. (C) The structure of the endogenous ligand (2’,3’-cGAMP) of STING (D) The strategy to be used to design the cyclic dinucleotide analogs by substitutions of the phosphodiester bonds in cGAMP with linker moieties. Examples of linkers to be used as substituents are shown in the inset.
The STING agonist 5,6-dimethylxanthenone-4-acetic acid (DMXAA) is a non-nucleotidic compound and showed potent anti-tumor activity in mouse models. However, DMXAA failed to clear clinical trial as it showed no potency for human STING activation (22). Recently, Zhang et al. reported three small-molecule screened from the ZINC database as STING agonist (23). These compounds showed potential STING activity and had antitumor activity in animal experiments. Amidobenzimidazole (ABZL) was reported by Ramanjulu et al., as STING agonist with anti-tumor activity in mouse (24). However, activity data of the compound on human-related experiments were not reported. Though significant progress has been made in developing small molecule STING agonists, currently, there are no non-nucleotidic STING agonists in clinical trials (2).
Innovation 
Computer-Aided Drug Design (CADD) methodologies have emerged as powerful tools in the arena of drug design and have been frequently used alongside experimental methods. The use of computational techniques could play a vital role in understanding the mechanism of STING activation and the discovery of STING agonists. To this end, Tsuchiya et al., (2016) studied the conformational changes at the c-terminal tail (CTT) of a STING monomer to understand the mechanism of activation of STING. However, the study was conducted on monomeric STING structure in water medium without considering the dimeric binding site as well as the lipid membrane environment of STING. With the literature regarding the structure and function of STING collected over the last decade and the availability of full-length cryo-EM structure of Apo-STING and cGAMP-STING (human), the computational study of STING presents a unique opportunity to develop small molecules that target the cGAS-STING-IRF3 signaling pathway.
In this spirit, this project has been designed to study the binding mechanism of STING and its endogenous ligand 2’, 3’-cGAMP and to design novel STING agonists that are resistant to hydrolysis. The hit compounds discovered from the study may be tested by experimental collaborator of Dr. Andreas Bender. Though the proposed experiments require extensive computational power, recent progress in hardware performance and in development of efficient MD simulation programs that utilize Graphics Processing Unit (GPU) have made MD simulation of macromolecules up to milliseconds feasible. The applicant proposed to use molecular docking and simulation, binding energy calculations and pharmacophore-based virtual screening which are methods popularly used in standard drug discovery. The applicant is trained in the area of drug discovery and has several published works. The study will be supervised by Dr. Andereas Bender who has accumulated expertise in academic research and pharmaceutical industry. Altogether, the proposed study can make a significant contribution to cancer immunotherapy and has a high chance of success within the expected time period of 3 years.
References:

1.	Hanahan, D. and Weinberg, R.A. (2011). cell, 144, 646-674.
2.	Marloye, M., Lawler, S.E. and Berger, G. (2019). Future Science.
3.	Gajewski, T.F., Schreiber, H. and Fu, Y.-X. (2013). Nature immunology, 14, 1014.
4.	Wu, J., Sun, L., Chen, X., Du, F., Shi, H., Chen, C. and Chen, Z.J. (2013). Science, 339, 826-830.
5.	Ablasser, A. (2019). Nature, 567.
6.	Barber, G.N. (2015). Nature Reviews Immunology, 15, 760-770.
7.	Abraham, M.J., Murtola, T., Schulz, R., Páll, S., Smith, J.C., Hess, B. and Lindahl, E. (2015). SoftwareX, 1, 19-25.
8.	Kumari, R., Kumar, R., Consortium, O.S.D.D. and Lynn, A. (2014). J. Chem. Inf. Model, 54, 1951-1962.
9.	Sun, H. (2008). Current medicinal chemistry, 15, 1018-1024.
10.	Shang, G., Zhang, C., Chen, Z.J., Bai, X.-c. and Zhang, X. (2019). Nature, 567, 389.
11.	Zhang, C., Shang, G., Gui, X., Zhang, X., Bai, X.-c. and Chen, Z.J. (2019). Nature, 567, 394.
12.	Li, L., Yin, Q., Kuss, P., Maliga, Z., Millán, J.L., Wu, H. and Mitchison, T.J. (2014). Nature chemical biology, 10, 1043.
13.	Shi, H., Wu, J., Chen, Z.J. and Chen, C. (2015).PNAS, 112, 8947-8952.
14.	Pronk, S., Páll, S., Schulz, R., Larsson, P., Bjelkmar, P., Apostolov, R., Shirts, M.R., Smith, J.C., Kasson, P.M. and Van Der Spoel, D. (2013). Bioinformatics, 29, 845-854.
15.	Koes, D.R. and Camacho, C.J. (2012). Nucleic acids research, 40, W409-W414.
16.	Gaulton, A., Bellis, L.J., Bento, A.P., Chambers, J., Davies, M., Hersey, A., Light, Y., McGlinchey, S., Michalovich, D. and Al-Lazikani, B. (2011). Nucleic acids research, 40, D1100-D1107.
17.	Ablasser, A. and Gulen, M.F. (2016). Journal of molecular medicine, 94, 1085-1093.
18.	Zhang, X., Shi, H., Wu, J., Zhang, X., Sun, L., Chen, C. and Chen, Z.J. (2013). Mol. cell, 51, 226-235.
19.	Gao, P., Ascano, M., Wu, Y., Barchet, W., Gaffney, B.L., Zillinger, T., Serganov, A.A., Liu, Y., Jones, R.A. and Hartmann, G. (2013). Cell, 153, 1094-1107.
20.	Jin, L., Hill, K.K., Filak, H., Mogan, J., Knowles, H., Zhang, B., Perraud, A.-L., Cambier, J.C. and Lenz, L.L. (2011). The Journal of Immunology, 187, 2595-2601.
21.	Liu, P., Sharon, A. and Chu, C.K. (2008). Journal of fluorine chemistry, 129, 743-766.
22.	Bibby, M., Phillips, R.M., Double, J. and Pratesi, G. (1991). British journal of cancer, 63, 57.
23.	Zhong, S., Li, W., Bai, Y., Wu, B., Wang, X., Jiang, S., Zhao, Y., Ren, J., Li, H. and Jin, R. (2019). PloS one, 14, e0216678.
24.	Ramanjulu, J.M., Pesiridis, G.S., Yang, J., Concha, N., Singhaus, R., Zhang, S.-Y., Tran, J.-L., Moore, P., Lehmann, S. and Eberl, H.C. (2018). Nature, 564, 439.        
        </p>   
    <figure>
  <img
  src="membrane.tif"
  alt="Top and bottom views of the Serotonin receptor 5HT2A.">
</div>

<center>
<footer class="flex-container footer">Copyright © 2021. Seke Keretsu. All rights reserved.
</footer>
</center>
</body>
</center>
</html>
