<html>
<head>
<title>Research</title>

    <meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="chrome=1">
<meta name="HandheldFriendly" content="True">
<meta name="MobileOptimized" content="320">
<meta name="viewport" content="width=device-width, initial-scale=1.0">
<meta name="referrer" content="no-referrer">
<meta name="description" content="Seke Keretsu.">

<style>
div {
  width: 100%;
  max-width: 1000px;
  min-width: 100px;
  background-color:#D3D3D3;
  text-align:justify;	
  text-indent:50px; 
  margin-right: 20px;
  margin-left:  20px;

}
</style>
</head>



<center>
<body>


<style>
.topnav {
  overflow: hidden;
  background-color: #333;
}

.topnav a {
  float: left;
  color: #f2f2f2;
  text-align: left;
  padding: 10px 8px;
  text-decoration: none;
  font-size: 17px;
}

.topnav a:hover {
  background-color: #ddd;
  color: black;
}

.topnav a.active {
  background-color: #4CAF50;
  color: white;
}

.illustration{
width: 70%
</style>


<div class="topnav">
  <a href="index.html">Home</a>
  <a class="active "href="research.html">Research</a>
  <a href="publication.html">Publication</a>
  <a href="software.html">Software</a>
  <a href="tutorial.html">Tutorial</a>
  <a href="contact.html">Contact</a>
  <a href="blog.html">Blog </a>
</div>
<div>
<br><p>
This includes ongoing research projects as well as other interest of mine that I inended to pursue in the near future. These works can be broadly classified into the application and development oriented projects </p>
<p>
This part will be updated from time to time. 
</p>
<hr>
<b>Computational study of serotonin receptors (GPCR): </b>
<br><br>
<center>
<img align='middle' class='illustration' style= 'max-width: 400px' src="membrane.jpg" alt="sarscov2">
</center>
<p>
Recent studies on GPCR proteins and CRISPR-CAS9 systems have demonstrated that GaMD techniques could be effectively used to study the binding and activation mechanisms of large biomolecular systems. I am curious to find out if these enhanced sampling techniques can be used to resolve existing structural enigmas in the areas of GPCR proteins such as Serotonin receptors.
</p>
<hr>
<b> Virtual screening of MEROPE (a Protease database) for poteintial COVID-19 main protease (SARS-CoV-2):</b><br><br>
<center>
<img align='middle' class='illustration' style= 'max-width: 400px' src="sarscov2.jpg" alt="sarscov2" height="500px">
</center>
<p>
In the fast-evolving coronavirus disease (COVID-19) pandemic, repurposing existing drugs and evaluating commercially available inhibitors against the druggable targets of the virus could be an effective strategy to accelerate the drug discovery process. The 3C-Like proteinase (3CLpro) of the severe acute respiratory syndrome coronavirus 2 (SARS-COV-2) has been identified as an important drug target due to its role in the viral replication. The lack of a potent 3CLpro inhibitor and the availability of the x-ray crystal structure of the 3CLpro (PDB-ID 6LU7) motivated us to perform computational study to identify commercially available potential inhibitors. A combination of modeling studies was performed to identify potential 3CLpro inhibitors from the protease inhibitor database MEROPS (https://www.ebi.ac.uk/merops/index.shtml).

We found 15 potential 3CLpro inhibitors with higher binding affinity than inhibitor in x-ray structure (α-ketoamide). Among them, Saquinavir (approved drug for HIV-1 treatment) and three other investigational drugs such as aclarubicin, TMC-310911, and Faldaprevir could be suggested as potential 3CLpro inhibitors.

</p>
<hr>

<b>
Modelling study of Nanobody binding to Angiotensin II Type 1 Receptor (AT1R) and Virtual Screening of Potential AT1R Inhibitors.
</b><br>
<p>
<b>Abstract:</b> Angiotensin II type 1 receptors (AT1R) have emerged as an important G Protein-Coupled Receptor (GPCR) because of the role it plays in regulating cardiovascular and renal activities. Consequently, inhibitors that target AT1R are considered as therapeutic agents for heart failure diseases. However, computational studies of the AT1R-G protein interactions have not been studied due to the oversized system that incurs high computational cost. This proposal was designed to study the coupling mechanism of the binding of ligand to AT1R and the conformational changes leading to the stabilization and activation of AT1R using the Gaussian accelerated molecular dynamics (GaMD) simulation. In addition, a docking based virtual screening protocol was also proposed to identify potential AT1R antagonists from the ZINC database. Binding free energy evaluation of selected ligands from the screening will be performed using Molecular Mechanics Poisson-Boltzmann Surface Area (MM-PBSA) method. The outcome of the study could provide useful insight into the mechanism of AT1R activation and may assist in the discovery of AT1R antagonists. 
    </p>
<hr>    
<b> Data Mining in PPI network:</b><br>

    <p>I have worked in the area of data mining in biological networks under the project entitled, “Protein complex identification in protein-protein interaction networks with the incorporation of gene expression profile,” supervised by Dr.Rosy Sarmah, Tezpur University. The study culminated in the development of new bioinformatics methods that showed statistically better protein complex prediction. The Java implementations of the methods are available at the links below:
http://agnigarh.tezu.ernet.in/~rosy8/shared.html or https://github.com/sekekeretsu/
        </p>
<hr>

<b> 
Mechanism of STING agonism:
</b>

<p>
The transmembrane protein stimulator of interferon genes (STING) is an important component of the immune signalling pathway and its activation has been shown to elevate immune response in tumour cancers. Hence, STING agonists are of vital pharmaceutical importance for cancer immunotherapy. My research proposal titled “Discovery of small molecules and dinucleotide analogs as STING agonist for cancer immunotherapy” was designed with the objective to study the mechanism of activation of STING and to develop small molecules and hydrolysis resistant dinucleotide STING agonists.  The study will involve protein modeling, molecular docking and simulation, binding energy calculations and virtual screening of chemical compounds, which are popularly used techniques in Computer–Aided Drug Discovery and can be carried out at the Data-Driven Drug Discovery and Molecular Informatics group. The study may also be carried out in collaboration with experimental partners. <a align='middle' class='illustration' style= 'max-width: 400px' href="sting.html"> ...More</a>
 

</p>
     
    <figure>
  <img align='middle' class='illustration' style= 'max-width: 400px'
  src="membrane.tif"
  alt="Top and bottom views of the Serotonin receptor 5HT2A.">
</div>

<center>
<footer class="flex-container footer">Copyright © 2021. Seke Keretsu. All rights reserved.
</footer>
</center>
</body>
</center>
</html>
